File: android/NeacyPlugin-master/app/src/main/java/com/neacy/router/MainActivity.java
package com.neacy.router;

import android.content.Intent;
import android.os.Bundle;
import android.support.design.widget.FloatingActionButton;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.Toolbar;
import android.view.Menu;
import android.view.MenuItem;
import android.view.View;

import com.neacy.annotation.NeacyCost;
import com.neacy.annotation.NeacyProtocol;

import neacy.neacymodule2.TestActivity;
import neacy.router.NeacyRouterManager;

@NeacyProtocol("Neacy://app/MainActivity")
public class MainActivity extends AppCompatActivity {


    @Override
    @NeacyCost("MainActivity.onCreate")
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
        setSupportActionBar(toolbar);


//        NeacyRouterManager.getInstance().init(this);

        NeacyRouterManager.getInstance().initRouter();

        FloatingActionButton fab = (FloatingActionButton) findViewById(R.id.fab);
        fab.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(MainActivity.this, TestActivity.class);
                startActivity(intent);

                // 如果想要调试class替换的话  打开注释 以及app/build.gradle中相对应的插件声明
//                neacy.neacymodule.LastLogger.printLog();
            }
        });
    }

    @Override
    public boolean onCreateOptionsMenu(Menu menu) {
        // Inflate the menu; this adds items to the action bar if it is present.
        getMenuInflater().inflate(R.menu.menu_main, menu);
        return true;
    }

    @Override
    public boolean onOptionsItemSelected(MenuItem item) {
        // Handle action bar item clicks here. The action bar will
        // automatically handle clicks on the Home/Up button, so long
        // as you specify a parent activity in AndroidManifest.xml.
        int id = item.getItemId();

        //noinspection SimplifiableIfStatement
        if (id == R.id.action_settings) {
            return true;
        }

        return super.onOptionsItemSelected(item);
    }
}

File: android/LottieFlicker/app/src/main/java/com/example/fcl/lottieflicker/MainActivity.kt
package com.example.fcl.lottieflicker

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.support.v7.widget.LinearLayoutManager
import kotlinx.android.synthetic.main.activity_main.recycler

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val data = arrayListOf(true, false, false, true, true, false, false, true,
            false, true, true, false, false, true,true, false, false, true,
            false, true, true, false, false, true)
        val adapter = MyAdapter(data)
        recycler.adapter = adapter
        recycler.layoutManager = LinearLayoutManager(this)

    }
}
